plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -1569,57 +1569,55 @@ $.extend(DataTable.prototype, UIControl.prototype, {
         var self = this;
 
         // higlight all columns on hover
-        $('td', domElem).hover(
-            function() {
-
-                if ($(this).hasClass('label')) {
-                    return;
-                }
+        $('td', domElem).hover(function() {
```

Comment:
Here actually nothing changed, reformatted code a little and made a variable re-usable.